src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,49 @@
-export const PeopleFilters = () => {
+import React from 'react';
+import classNames from 'classnames';
+import { Link, useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../utils/searchHelper';
+import { SearchLink } from './SearchLink';
+
+export const PeopleFilters: React.FC = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const allCenturies = ['16', '17', '18', '19', '20'];
```

Comment:
move this variable outside the component

Reply:
fixed